pkg/clustertask/clustertask.go

```diff
@@ -30,6 +30,24 @@ import (
 
 var clustertaskGroupResource = schema.GroupVersionResource{Group: "tekton.dev", Resource: "clustertasks"}
 
+func GetAllClusterTaskNames(p cli.Params) ([]string, error) {
+	cs, err := p.Clients()
+	if err != nil {
+		return nil, err
+	}
+
+	ps, err := List(cs, metav1.ListOptions{})
```

Comment:
Probably a better var name here would be `cts` or just more explicitly `clustertasks`.